### ISSUE TITLE
Arcade Online appears wrong enemies

### ISSUE THREAD

**Issue by Viniciaao** [CableDorado2/Ikemen-Plus-Ultra#11]
I wanted to know if you ever intend to fix this, where different enemies appear for each player in online arcade mode

**Comment by CableDorado2** [comment]
Yeah is in the plans, we have to see how to synchronize the random tables that are generated for both players. Sometimes they are synchronized and other times they are not. Maybe we should generate a single table for the entire session.

Although I must clarify that if the characters use the arcade path 1, 2, (...) parameter in select.def they will always be synchronized.

**Comment by Viniciaao** [comment]
Taking advantage of your answer, why in this version of mugen do I suffer
from lag and in other versions this doesn't happen?

Em sáb., 17 de fev. de 2024 4:52 PM, CD2 ***@***.***>
escreveu:

**Comment by Viniciaao** [comment]
What do you mean by "Other versions" it's about other Mugen created or
modified by others, understand?

Em sáb., 17 de fev. de 2024 5:37 PM, Viniciaao ***@***.***>
escreveu:

**Comment by CableDorado2** [comment]
Yes, are optimization problems with Simple DirectMedia Layer library, we hope done some things about that in final updates planned.
